src/vfs/syspath_win32.c

```diff
@@ -64,10 +63,12 @@ static void _vfs_set_error_win32(const char *file, int line) {
 static VFSInfo vfs_syspath_query(VFSNode *node) {
     VFSInfo i = {0};
 
-    if(!PathFileExists(node->_wpath_)) {
+    SDL_RWops *rwops = SDL_RWFromFile(node->_path_, "rb");
```

Comment:
Maybe there's a better option? Opening a file may fail due to many reasons, even though the file actually exists.

Reply:
I was following [this thread](https://stackoverflow.com/questions/2903360/how-to-check-whether-a-file-exists-in-c-drive-using-vc) - I couldn't get shlwapi to link, stat wouldn't be available for VC++ users, fopen is marked as deprecated by SDL2 which suggests using its own functions.

Reply:
I've just pushed 85c4fb8, which fixes the linking error for me. Can you please check if that works for you as well?

I'm pretty sure that VC++ users won't be able to compile Taisei anyway without some drastic changes. Taisei is written in (mostly standard) C11; as far as I know, VC++ doesn't have good support for it. Our build system also currently expects a GCC or clang -compatible compiler frontend, due to all the hardcoded flags.

That said, stat would most likely not work with non-ASCII paths, anyway.

Reply:
I am too lazy to use toolchain file, so I'm running the same flags as before. But now, with fresh CMakeCache.txt I'm hitting new kind of error altogether...
```
x86_64-w64-mingw32-gcc: error: unrecognized command line option ‘-rdynamic’
```

I'm too tired of trying to make cmake work, I will rather wait for meson before further testing.

By the way, I found out that I have unistd.h in mingw headers because of Arch Linux `mingw-w64-headers` package, which is required by `mingw-w64-gcc`.

Reply:
I can sympathize, CMake *is* a world of misery and pain. That's why we're moving to Meson in the first place. But hey, at least it's not autohell ;)

`unistd.h` is expected to be in the mingw headers, it shouldn't define `_POSIX_VERSION` though.